storage/blockchain/src/service/mod.rs

```diff
@@ -21,10 +21,6 @@
 //!
 //! The 2nd allows any caller to send [`WriteRequest`][req_w]s.
 //!
-//! The `DatabaseReadHandle` can be shared as it is cheaply [`Clone`]able, however,
-//! the `DatabaseWriteHandle` cannot be cloned. There is only 1 place in Cuprate that
-//! writes, so it is passed there and used.
-//!
```

Comment:
### Misc
Not true anymore, we need `Clone` on `CupratedRpcHandler` and thus the `DatabaseWriteHandle`.

Reply:
This does need to be removed for the tx-pool handle, however the database is still only written to in 1 place, the blockchain manager.

You shouldn't need to hold a write handle to the blockchain in the RPC